+nigeLab/@Animal/init.m

```diff
@@ -17,8 +17,8 @@
 
 %GET BLOCKS
 % Remove other folder names
-Recordings = dir(fullfile(animalObj.RecDir));
-Recordings = Recordings(~ismember({Recordings.name},{'.','..'}));
+Recordings = cellfun(@(x) dir(fullfile(animalObj.RecDir,['*' x])),supportedFormats,'UniformOutput',false);
+Recordings = cat(1,Recordings{:});
```

Comment:
@Nabarb thinking about this more, it's incorrect: Animal/init should be initialized from the Animal folder. This part is looking for Recordings, but there is a whole bunch of parsing at Block level that actually checks the Recordings filename, so this should be reverted.

Reply:
Yes, but as far as I know we don't check for the extension anywhere if a block is created during the init step of animal. I run into the issue of having a .nex file with the same name a .rhs one, and Nigel was ending up creating an invalid Block targeting that file. We can do it differently but we should add that additional check on the extension.

Reply:
I see, makes sense